.prettierignore

```diff
@@ -3,3 +3,4 @@
 blog
 .docusaurus
 .github
+docs/node/*.md
```

Comment:
I wouldn't do that, this applies to all of the markdown not just code samples

Reply:
Does our prettier config modify markdown, not just code samples? I might have missed something here but it seems like prettier just formats code samples in this repo.

Reply:
I’m pretty sure it does, that should be easy to confirm

Reply:
I confirmed that prettier does in fact modify markdown in addition to code samples: https://prettier.io/blog/2017/11/07/1.8.0.html .

To avoid prettier mangling our TS samples, we need to add `<!-- prettier-ignore -->` before each example.

Reply:
This PR gets prettier doing single quotes and bracket spacing so you don't need to ignore: https://github.com/temporalio/documentation/pull/573